app/src/main/res/values-sw600dp-land/refs.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this the right config do we think? Maybe just values-w600dp, which would also pick up very large portrait screens, which is fine.

Why is this in a values ref rather than just directly having the two layouts in layout/ and layout-w600dp/ ?